src/app/wallets/get-on-chain-fee.ts

```diff
@@ -1,35 +1,49 @@
-import { TxDecoder } from "@domain/bitcoin/onchain"
-import { AccountsRepository, WalletsRepository } from "@services/mongoose"
-import { WithdrawalFeeCalculator } from "@domain/wallets"
-import { OnChainService } from "@services/lnd/onchain-service"
 import { BTC_NETWORK, getOnChainWalletConfig } from "@config"
 import { checkedToSats, checkedToTargetConfs, toSats } from "@domain/bitcoin"
+import { TxDecoder } from "@domain/bitcoin/onchain"
 import {
   CouldNotFindError,
-  LessThanDustThresholdError,
   InsufficientBalanceError,
+  LessThanDustThresholdError,
 } from "@domain/errors"
+import { checkedToWalletId, WithdrawalFeeCalculator } from "@domain/wallets"
 import { LedgerService } from "@services/ledger"
+import { OnChainService } from "@services/lnd/onchain-service"
+import { WalletsRepository } from "@services/mongoose"
 
 const { dustThreshold } = getOnChainWalletConfig()
 
 export const getOnChainFee = async ({
-  wallet,
+  walletId,
+  withdrawFee,
   amount,
   address,
   targetConfirmations,
 }: GetOnChainFeeArgs): Promise<Satoshis | ApplicationError> => {
-  const withdrawalFeeCalculator = WithdrawalFeeCalculator()
+  const amountChecked = checkedToSats(amount)
```

Comment:
I don't like that we are naming the 'correct' type `amountChecked` and the raw types `amount` it should be the other way around. Correct types should be the norm and have the simpler name `amount` - unchecked are the special case and should have the special name `amountRaw`.


This is a general observation - not just relating to this PR.

Reply:
that is a good point. this would also bring shorter variable name within the function, which would avoid some line return/make the code more compact.